be/src/exec/pipeline/scan/scan_operator.cpp

```diff
@@ -100,31 +112,47 @@ bool ScanOperator::has_output() const {
     if (!_get_scan_status().ok()) {
         return true;
     }
-    if (has_buffer_output()) {
+
+    // Try to buffer enough chunks for exec thread, to reduce scheduling overhead.
+    // It's like the Linux Block-Scheduler's Unplug algorithm, so we just name it unplug.
```

Comment:
Is decoupling word more comment and understandable  than unplug?  linux 's users are developers, but our users are normal users.  Especially you expose the metric `BufferUnplugCount` to our user.